arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/transients/DeploymentInfo.java

```diff
@@ -18,12 +18,14 @@
 import java.util.HashMap;
 import java.util.Map;
 
+import org.apache.tuweni.bytes.Bytes;
 import org.hyperledger.besu.datatypes.Address;
 
 /** Stores information relative to contract deployment. */
 public class DeploymentInfo {
   private final Map<Address, Integer> deploymentNumber = new HashMap<>();
   private final Map<Address, Boolean> deploymentStatus = new HashMap<>();
+  private final Map<Address, Bytes> initializationCodes = new HashMap<>();
```

Comment:
We need to remember the initialization code so that we can query its `codeSize` for AccountFragments pertaining to accounts currently undergoing deployment.